src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/runInTerminalToolTelemetry.ts

```diff
@@ -97,9 +97,13 @@ export class RunInTerminalToolTelemetry {
 		pollDurationMs?: number;
 		terminalExecutionIdleBeforeTimeout?: boolean;
 		exitCode: number | undefined;
-		autoReplyCount?: number;
 		inputUserChars: number;
 		inputUserSigint: boolean;
+		inputToolManualAcceptCount?: number;
+		inputToolManualRejectCount?: number;
+		inputToolManualChars?: number;
+		inputToolSuggestChars?: number;
+		inputToolSuggestInputCount?: number;
```

Comment:
Think we want to keep the "auto" naming rather than "suggest", since all `inputTool*` ones are suggestions, just they're either manually presented to the user or automatically actioned.

Reply:
I misunderstood, we can remove these I think?